docs/source/usage/security_recommendations.rst

```diff
@@ -112,5 +112,203 @@ Note:
+
+The **preferred way** of restoring iDRAC to a known good state is to export a system configuration mold (JSON settings file) before leasing a node and import the configuration mold to restore the iDRAC settings after the lease ends. Two ways to import/export iDRAC settings:
+
+* Import/Export iDRAC configuration with REST APIs using Redfish.
+* Import/Export iDRAC configuration OpenStack Ironic Bare Metal Cleaning service.
+
+Configuration molds currently support only the OEM settings section that uses the iDRAC Server Configuration Profile (SCP) and can be edited as necessary if it complies with the SCP. The mold can be stored in either a Swift container or an HTTP server that has support for 'PUT' requests. More information about SCP: `SCP_Reference_Guide`_.
+The following is an example of a configuration mold:
+
+
+.. prompt:: bash $
+
+  {
+  "oem": {
+    "interface": "idrac-redfish",
+    "data": {
+      "SystemConfiguration": {
+        "Model": "PowerEdge R640",
+        "ServiceTag": "8CY9Z99",
+        "TimeStamp": "Fri Jun 26 08:43:15 2020",
+        "Components": [
+          {
+            [...]
+            "FQDD": "NIC.Slot.1-1-1",
+            "Attributes": [
+              {
+              "Name": "BlnkLeds",
+              "Value": "15",
+              "Set On Import": "True",
+              "Comment": "Read and Write"
+              },
+              {
+              "Name": "VirtMacAddr",
+              "Value": "00:00:00:00:00:00",
+              "Set On Import": "False",
+              "Comment": "Read and Write"
+              },
+              {
+              "Name": "VirtualizationMode",
+              "Value": "NONE",
+              "Set On Import": "True",
+              "Comment": "Read and Write"
+              },
+            [...]
+            ]
+          }
+        ]
+      }
+    }
+  }
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+The `iDRAC`_ hardware type requires the python-dracclient and the sushy library to be installed on the ironic conductor node(s) if the node is configured to use an idrac-redfish or idrac-wsman interface implementation.
+
+* Install python-dracclient and sushy library:
+
+.. prompt:: bash $
+
+  sudo pip install 'python-dracclient>=3.1.0' 'sushy>=2.0.0'
+
+The iDRAC driver supports WSMAN for the bios, inspect, management, power, raid, and vendor interfaces. In addition, it supports Redfish for the bios, inspect, management, power, and raid interfaces. The iDRAC driver allows you to mix and match WSMAN and Redfish interfaces.
+
+* Enable the iDRAC hardware type with the required interfaces idrac-wsman
+
+.. prompt::
+
+  [DEFAULT]
+  enabled_hardware_types=idrac
+  enabled_management_interfaces=idrac-wsman
+  enabled_power_interfaces=idrac-wsman
+  enabled_vendor_interface=idrac-wsman
+  enabled_bios_interface=idrac-wsman
+
+* To enable all optional features (BIOS, inspection, RAID, and vendor passthru) using Redfish where it is supported and WSMAN where not, use the following configuration:
```

Comment:
Maybe also mention that ironic_conductor service needs a restart to take effect.

Reply:
Done!